src/Components/WebAssembly/Authentication.Msal/src/Models/MsalLoginMode.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Authentication.WebAssembly.Msal.Models
+{
+    public enum MsalLoginMode
+    {
+        Popup,
+        Redirect
+    }
```

Comment:
Can we change this to be string based? I'd rather not have us use enums for interop since it makes debugging harder, or use the JsonConverter attribute on the property to turn them into strings when passed to JS

Reply:
No problem, I'll make the change.